### ISSUE TITLE
Fluorine atom unknown?

### ISSUE THREAD

**Issue by ricalessandri** [jbarnoud/cgbuilder#5]
The molecule 2,2,2-trifluoroethanol shows up like this:
<img width="310" alt="TFEOL_cgbuilder" src="https://user-images.githubusercontent.com/10633291/57312666-4f82a480-70ee-11e9-84e0-f78271553b03.png">
Maybe something to do with the not-so-common fluorine atoms?

Coordinates of the molecule:
```
LIGPARGEN GENERATED GRO FILE
    9
    1UNK    O00    1   0.100   0.100   0.000
    1UNK    C01    2  -0.043   0.100   0.000
    1UNK    C02    3  -0.107   0.100   0.140
    1UNK    F03    4  -0.016   0.108   0.239
    1UNK    F04    5  -0.192   0.205   0.153
    1UNK    F05    6  -0.179  -0.013   0.159
    1UNK    H06    7   0.129   0.100  -0.093
    1UNK    H07    8  -0.076   0.012  -0.055
    1UNK    H08    9  -0.076   0.189  -0.055
   1.00000   1.00000   1.00000
```

**Comment by jbarnoud** [comment]
I could load the structure from an SDF file. It seems that NGL does not manage to guess the element from the atom name for fluorine.

**Comment by ricalessandri** [comment]
I see. I am not familiar with that file format. Just thinking out loud: is an on-the-fly conversion from a user-uploaded pdb/gro to SDF or any other format that NGL "likes better" as a pre-processing step something reasonable at all?

**Comment by jbarnoud** [comment]
Ideally, NGL should manage your input file. I'll see if I can convince it. In the meantime, any file format that is explicit about atom elements will likely do better. I expect that a well-formated PDB (with the element at the end of the line) or a MOL2 file would likely work.

**Comment by ricalessandri** [comment]
Right. I'm using LigParGen's PDBs, which look like this: https://github.com/ricalessandri/Martini3-small-molecules/blob/main/models/AA/oplsaa-LigParGen/BZTF_LigParGen.pdb, i.e., don't contain the element at the end of the line.

Will keep this in mind for the future, and perhaps I should opt for converting those PDBs to better formatted ones.
